ambry-replication/src/test/java/com.github.ambry.replication/ReplicationTest.java

```diff
@@ -309,6 +321,182 @@ public void replicationPauseTest() throws Exception {
     }
   }
 
+  /**
+   * Tests that replication between a local and remote server who have different
+   * blob IDs for the same blobs (via StoreKeyConverter)
+   * @throws Exception
+   */
+  @Test
+  public void replicaThreadTestConverter() throws Exception {
+    MockClusterMap clusterMap = new MockClusterMap();
+    Pair<Host, Host> localAndRemoteHosts = getLocalAndRemoteHosts(clusterMap);
+    Pair<Host, Host> expectedLocalAndRemoteHosts = getLocalAndRemoteHosts(clusterMap);
+    Host localHost = localAndRemoteHosts.getFirst();
+    Host remoteHost = localAndRemoteHosts.getSecond();
+    Host expectedLocalHost = expectedLocalAndRemoteHosts.getFirst();
```

Comment:
what are expected local and expected remote hosts?

Reply:
The method I had available generates the hosts in local+remote pairs.  I really only need the expectedLocalHost part of it.  I use it to compare the state of the local host after testing to ensure that local host is in the expected state.